client/my-sites/plugins/plugin-management-v2/plugin-action-status/render-status-message.tsx

```diff
@@ -0,0 +1,180 @@
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import { ReactElement, ReactNode } from 'react';
+import {
+	ACTIVATE_PLUGIN,
+	DEACTIVATE_PLUGIN,
+	DISABLE_AUTOUPDATE_PLUGIN,
+	ENABLE_AUTOUPDATE_PLUGIN,
+	REMOVE_PLUGIN,
+} from 'calypso/lib/plugins/constants';
+import {
+	PLUGIN_INSTALLATION_COMPLETED,
+	PLUGIN_INSTALLATION_ERROR,
+	PLUGIN_INSTALLATION_IN_PROGRESS,
+} from 'calypso/state/plugins/installed/status/constants';
+import { CurrentSiteStatus, PluginActionStatus, PluginActionStatusMessage } from '../types';
+import type { SiteDetails } from '@automattic/data-stores';
+
+import './style.scss';
+
+interface Props {
+	selectedSite?: SiteDetails;
+	groupKey: PluginActionStatus;
+	groupedStatues: { [ key in PluginActionStatus ]: Array< CurrentSiteStatus > };
+	hasOneStatus: boolean;
+}
+
+const getStatusMessage = (
```

Comment:
Do we want to rename this to plural (messages) as it indeed returns multiple messages?

Reply:
This was returning a single message only, but I have refactored the code and now it returns multiple messages